tests/cupy_tests/test_cusolver.py

```diff
@@ -177,3 +178,39 @@ def test_syevjBatched(self):
 
         self.assertEqual(v.shape, a.shape)
         self.assertEqual(w.shape, a.shape[:-1])
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64, numpy.complex64, numpy.complex128],
+    'shape': [(32, 32), (33, 32)],
+    'nrhs': [None, 1, 4],
+    'compute_type': [None,
+                     _linalg.COMPUTE_TYPE_FP16,
+                     _linalg.COMPUTE_TYPE_TF32,
+                     _linalg.COMPUTE_TYPE_FP32],
+}))
+@attr.gpu
+class TestGels(unittest.TestCase):
+    _tol = {'f': 1e-5, 'd': 1e-12}
```

Comment:
Does `_tol` also depend on `compute_type`?

Reply:
Theoretically, there is no need to change `_tol` depending on `compute_type`.